jerry-ext/include/jerryscript-ext/arg.h

```diff
@@ -90,7 +90,61 @@ typedef enum
   JERRYX_ARG_REQUIRED
 } jerryx_arg_optional_t;
 
+/**
+ * Indicates the rounding policy which will be choose to transform a integer.
+ */
+typedef enum __attribute__((packed))
```

Comment:
Having read the discussion, I'm still not convinced about the need for a public option type.
- In the discussion, the option type was planned to contain rounding, clamping, optionality, requiredness - but eventually it just remained with rounding and clamping. Optionality and requiredness are still passed as regular args. So, the functions already take 4 arguments. How much longer is a 5-args signature, when does it count as too long?
- The empty initializer is not C but C++, as also mentioned in the discussion. How better is `{0}` than `0, 0`? (Provided that you'd like to use 0-based defaults instead of writing out the enum values explicitly.)
- I'm not sure that accessing fields of a struct of packed enums is in any ways different from doing the shifting-masking manually (at the machine code level). But if we want to avoid bit ops, we can still have a non-public struct of two uint8_t fields that contain the enum values. Then, at implementation source code level, we only have to write out casting explicitly (to enum and uint8_t), and the compiler will hide the packing.